test/odbc/psqlodbc/test/time.cpp

```diff
@@ -139,12 +139,23 @@ TEST_F(PSQL_DataTypes_Time, Insertion_Fail) {
     "00:60:0",    // Minute over range
     // "00:00:60",   // Second over range
     "00-00-00",   // Format hh-mm-ss
-
-    // ODBC API format for Time is hh:mm:ss
+    "0000-12-31",     // Under Minimum
+    // "10000-01-31",    // Over Maximum *BBF Valid, PG Invalid
+    // ODBC API format for Time is hh:mm:ss (as well as YYYY-MM-DD in BBF)
     // Below are valid in SQL Server
-    "00:00:00.00 AM"
-    "00AM"
-    "00 AM"
+    "00:00:00.00 AM",
+    "00AM",
+    "00 AM",
+    // "1/31/00",        // m/dd/yy *BBF Valid on Ubuntu 20.x, Invalid on Ubuntu 22.x & PG Ubuntu 20.x
```

Comment:
why is behaviour different on Ubuntu 22.x? Does same apply to SQL server as well